.circleci/config.yml

```diff
@@ -71,7 +71,7 @@ workflows:
             branches:
               only:
                 - master
-                - valider-openam-tokens
+                - feature-toggle-endepunkt
```

Comment:
Hei!

Husk å fjerne denne før du merger til master. Ellers kan den ligge der i lang tid før vi oppdager den. Det er det vi kaller for teknisk gjeld. :)

Mvh.
Kjetil Svalestuen
Utvikler i Bekk

Reply:
Hei,

Takk for oppmerksomheten.
Dette er nå fikset.

Venlige hilsener,
André "Pølsa" Mossige
Utvikler i Bekk